src/NCI.OCPL.Api.Glossary/Controllers/TermsController.cs

```diff
@@ -43,15 +58,48 @@ public Task<GlossaryTerm> GetById(string dictionary, AudienceType audience, stri
             if (language.ToLower() != "en" && language.ToLower() != "es")
                 throw new APIErrorException(404, "Unsupported Language. Please try either 'en' or 'es'");
 
-            return _termsQueryService.GetById(dictionary, audience, language, id);
+            if (useFallback == false) {
+                return await _termsQueryService.GetById(dictionary, audience, language, id);
+            }
+            // Implement fallback logic.
+            // 1) Terms Dictionary and Patient
+            // 2) Terms Dictionary and Health Professional
+            // 3) Empty Dictionary and Patient
+            // 4) Empty Dictionary and Health Professional
+            // 5) Genetics Dictionary and Patient
+            // 6) Genetics Dictionary and Health Professional
+            // Depending on the given Dictionary and Audience inputs, loop through the fallback logic
+            // options until a term is found. If none of the options return a term, then throw an error.
+            else {
+                Tuple<string, AudienceType> currentValue = new Tuple<string, AudienceType>(dictionary, audience);
+                LinkedListNode<Tuple<string, AudienceType>> start = _fallbackOptions.Find(currentValue);
+                LinkedListNode<Tuple<string, AudienceType>> current = start;
```

Comment:
I think I see what you're trying to do here, but it's not a matter of starting at different place in the order.  The first definition to find should have the same audience as the summary the lookup started from.

Reply:
I let @sarinapadilla know that I am good with having a HP fallback list and a Patient fallback list. Just make it 2 read only vars, and the TermsController should pick the right one based on the audience.

Any future complexities can introduce refactoring into a more scalable solution then... ;)